engine/components/sql/src/main/scala/pl/touk/nussknacker/sql/service/DatabaseLookupEnricher.scala

```diff
@@ -42,14 +38,29 @@ object DatabaseLookupEnricher {
 
 class DatabaseLookupEnricher(dBPoolConfig: DBPoolConfig) extends DatabaseQueryEnricher(dBPoolConfig) {
 
+  protected def tableParam(): Parameter = {
+    val schemaMetaData = try {
+      dbMetaDataProvider.getSchemaDefinition()
+    } catch {
+      case e: Exception => SchemaDefinition.empty()
```

Comment:
Let's at least log sth? :) And maybe use scalas: ` case NonFatal(e) => `

Reply:
Added